### PR TITLE
Add a wallet-proxy load simulator.

### PR DESCRIPTION
**Pull request by abizjak**
## Purpose

Add a testing utility for load testing the wallet-proxy.

### CODE REVIEW

File: wallet-proxy-load-simulator/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+example invocation will thus look like
+```console
+wallet-proxy-load-simulator --wp-url http://wallet-proxy.stagenet.concordium.com --accounts accounts.json --delay 100 --max-parallel 100
+```
+
+The tool outputs to `stdout`. Each line contains the 
+- `i`, the identifier of the worker that sent the request
+- `url` the path of the URL that was queried
+- `diff` the time in milliseconds until the server responded
+- `success`, whether the request was successful, i.e., whether the server
```
Comment (EmilLa1) [inline wallet-proxy-load-simulator/README.md:30]: Why not just print the http status code? I would guess that sometimes the more fine grained result could become beneficial?

**Review by EmilLa1 — APPROVED** [review]
looks good. One minor comment

File: wallet-proxy-load-simulator/README.md (hunk 1 above)
Comment (abizjak) [reply wallet-proxy-load-simulator/README.md]: No reason, did not think it was super useful. Changed now.

**Review by mh-concordium — APPROVED** [review]
No comments.

**Pull request merged**